View3D/Utility/NetObjectExtension.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+public static class ObjectExtension
+{
+    public static T Copy<T>(this T lObjSource)
```

Comment:
Is this needed or could the object be manually copied? 
In my experience approaches like this always have some corner cases which makes hard to track down bugs. 
Unless there is a need to deep copy loots of simple objects, please avoid it. 
Consider looking into record types

Reply:
removed in b6f70ede5e1795c0fe14e724731700cb136ffe2b